sdk/core/core-client/review/core-client.api.md

```diff
@@ -203,6 +203,7 @@ export interface OperationArguments {
 export interface OperationOptions {
     abortSignal?: AbortSignalLike;
     requestOptions?: OperationRequestOptions;
+    responseMetadata?: RawResponseCallback;
```

Comment:
in Python this is called the `raw_response_hook`, which I don't think is a good name for us to use, but I don't think `responseMetadata` quite captures it either. 

Perhaps we could re-use your type name for it and call the property `rawResponseCallback`?

Reply:
oops, forgot to rename the property.